src/components/SideBar/ItemDrawer/ItemDrawer.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import { Header, Grid } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Drawer from 'components/Drawer'
+import AssetCard from 'components/AssetCard'
+import { Props, State } from './ItemDrawer.types'
+import './ItemDrawer.css'
+
+export default class ItemDrawer extends React.PureComponent<Props, State> {
+  state = {
+    listMode: false
+  }
+
+  renderGrid() {
+    const { items } = this.props
+    const { listMode } = this.state
+    let el = []
+
+    for (let i = 0; i < items.length; i += 3) {
+      let row = []
+
+      for (let j = i; j < i + 3; j++) {
+        const item = items[j]
+        if (!item) break
+
+        row.push(
+          <Grid.Column>
+            <AssetCard assetId={item.assetId} title={item.title} thumbnail={item.thumbnail} horizontal={listMode} />
```

Comment:
Related to comment [here](https://github.com/decentraland/builder/pull/30/files#diff-5277caf73452838c600d929235edbecaR17)